moonshot/src/redteaming/attack/context_strategy.py

```diff
@@ -93,6 +92,8 @@ def process_prompt_cs(
         endpoint_id,
         num_of_previous_chats: int,
     ) -> str:
+        from moonshot.src.redteaming.session.chat import Chat
+
```

Comment:
hmm is there a reason to putting this import only in this portion of the code?

Reply:
this is ensure there's no cyclic dependency in context strategy and chat. thinking if there's a better way to do this as well

Reply:
might need to rethink how come got cyclic dependency. like chat need context strategy and context strategy needs chat